doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -107,7 +107,10 @@ Bluetooth® LE
 Bluetooth Mesh
 --------------
 
-|no_changes_yet_note|
+* Updated:
+
+  *  Metadata has been added to the Light Lightness Server, Light HSL Server, Light CTL Temperature Server, the Sensor Server, and Time Server models in form of an optional parameter.
+     Usage is demonstrated in the light_ctrl sample.
```

Comment:
I think it is worth to add reference to sample documentation here.

Reply:
Needs `:ref:` link.

Reply:
I'll add a sentence under ´Bluetooth samples´ section.